scripts/db_migrator.py

```diff
@@ -174,6 +174,21 @@ def migrate_copp_table(self):
         for copp_key in keys:
             self.appDB.delete(self.appDB.APPL_DB, copp_key)
 
+    def migrate_feature_table(self):
+        '''
+        Combine CONTAINER_FEATURE and FEATURE tables into FEATURE table.
+        '''
+        feature_table = self.configDB.get_table('FEATURE')
+        for feature, config in feature_table.items():
+            state = config.pop('status', 'disabled')
```

Comment:
So if the status is not known, the default value for the feature after migration will be `state=disabled`. Has this been discussed/decided? Just curious.

Reply:
My take is that the feature is not enabled in init_cfg.json, then it would be default as disabled. But existing config will take effect. so if it is already enabled, it stays enabled.